docs/modules/ROOT/pages/documentation/payara-server/logging/log-to-console.adoc

```diff
@@ -0,0 +1,36 @@
+[[log-to-console]]
+= Log To Console
+
+The `Log to Console` option controls if the server writes the logging entries to the console. The property only has an effect when the server or instance is started in verbose mode.
+
+[[enable]]
+== Enable Logging To Console
+
+To enable the `Log to Console` option you simply have to start the Server or Instance with the verbose option.
+
+[source, shell]
+----
+asadmin start domain --verbose
+----
+
+When the server or domain is not started in verbose mode, the `java.util.ConsoleHandler` will not be active for logging although it is defined as handler in the _logging.properties_ file because there is no console attached with the process running the Payara server or Instance.
+
+When in verbose mode, the log to Console option will also be active, regardless of the initial value in the _logging.properties_ as otherwise, no log output would be visible on the console. Once started in verbose mode, you can use the Admin Console or the asadmin CLI to change the option.
+
+[[using-web-admin-console]]
+== Configure Logging To Console Using Admin Console
+
+To configure `Log to Console` option using Admin Console:
+
+image:logging/log_to_file.png[Log to Console disabled]
+
+[[using-asadmin-utility]]
+== Configure Logging To File Using Asadmin CLI
```

Comment:
I've fixed the typo (File -> Console) in this commit after merging this PR: https://github.com/payara/Payara-Community-Documentation/commit/b6520676f0995a9d55f3562badd5082188199693